orchestrator.py

```diff
@@ -24,7 +24,7 @@ def get_items_data(db_queries):
                 item_id = uin[0]
                 items_to_store_and_notify.append((item_id, price))
             else:
-                out_of_stock_items.append(uin)
+                out_of_stock_items.append((uin[0],))
```

Comment:
you can save the `item_id` before the if and use is where ever you need it

Reply:
changed!